ironfish-rust-nodejs/src/structs/note_encrypted.rs

```diff
@@ -9,20 +9,31 @@ use napi::bindgen_prelude::*;
 use napi::JsBuffer;
 use napi_derive::napi;
 
+use ironfish_rust::merkle_note::NOTE_ENCRYPTION_KEY_SIZE;
+use ironfish_rust::note::ENCRYPTED_NOTE_SIZE;
 use ironfish_rust::MerkleNote;
 
 use crate::to_napi_err;
 
 #[napi]
-pub const ENCRYPTED_NOTE_LENGTH: u32 = 296;
+pub const NOTE_ENCRYPTION_KEY_LENGTH: u32 = NOTE_ENCRYPTION_KEY_SIZE as u32;
+
+#[napi]
+pub const ENCRYPTED_NOTE_PLAINTEXT_LENGTH: u32 = ENCRYPTED_NOTE_SIZE as u32;
+
+#[napi]
+pub const MAC_LENGTH: u32 = 16;
```

Comment:
We do have this number defined here: https://github.com/iron-fish/ironfish/blob/master/ironfish-rust/src/serializing.rs#L86

Up to you whether you want to move it or anything, though

Reply:
Yeah I saw that, but serializing and aead are not public, so I created a new one. Should I make the serializing and aead public?

Reply:
It makes sense to me to make those public, I don't think we care too much about whether modules are public vs private in Rust. I'm okay with leaving it there, I don't have any particular ideas for where to move it to right now

Reply:
Reused the mac size defined in aead